### DISCUSSION TITLE
Question regarding design standard for bilingual emails

### DISCUSSION THREAD

**Discussion by ValerieCle** [gc-da11yn/gc-da11yn.github.io#571] [Q&A] [answered]
When writing bilingual emails, what is the better option to display both official language? 

**(1) Side-by-side: the official languages appear next to each other**
i.e : 
Bonjour/Hello

Nous tenons à vous informer que [Nom de l'utilisateur] n'a plus besoin d'accès aux systèmes suivants/Please be informed that [Name of User] no longer requires access to the following system(s)

**(2) Over‑and‑under format: the official languages are displayed one above the other**
ie.: 

An English message follows.

Bonjour,

Nous tenons à vous informer que [Nom de l'utilisateur] n'a plus besoin d'accès aux systèmes suivants :

☐ A
☐ B
☐ C

Notre équipe a confirmé leur départ de l'Inscription selon les informations fournies par [Nom du gestionnaire].

Merci,

________________________________________________________________________________________________________________________________________________________________________

Hello,

Please be informed that [Name of User] no longer requires access to the following system(s): 

☐ A
☐ B
☐ C

Our team has confirmed their departure from Registration as per the information provided by [Manager Name]. 

Thank you,

**Comment by lynelal** [comment]
M365 has too many issues related to the _Official Languages Act_ and the _Accessible Canada Act_. The people most affected are **people who are bilingual**, that is they read and write content in both official languages. I believe emails and all other file types need to be unilingual.

**Comment by kimviens** [comment, marked as answer]
Hi @ValerieCle. 
The [Making Accessible Emails](https://a11y.canada.ca/en/making-accessible-emails/#bilingual-message-if-applicable) guide has a section about bilingual emails (I will add that I recommend you dont put the messages side by side but one message first and the second after):

### Bilingual message (if applicable)
Whenever possible, avoid creating bilingual emails. Bilingual emails can introduce a lot of accessibility and usability challenges for a wide range of users.

If you must use bilingual emails, be sure to make them as accessible as possible.

- [Add a bookmark](https://a11y.canada.ca/en/making-accessible-emails/#bookmarks) to let users skip to the section containing their language of choice.
- Include location of language information.
  - in emails where English is the first language used, include “Le français suit l’anglais” above the English content
  - in emails where French is the first language used, include “English follows” above the French content

**Setting a language**
If you set an entire document to English but there are some sections that have text written in French, English screen readers will try to pronounce the French information, resulting in gibberish sounds.

- Perform language markup to help screen readers read French text properly. Use this for changing the proofing language of sections of a document 
  1.  Set **Proofing Language** (Alt, V, U, L). 
  2. Highlight the text that you want set to a different language. 
  3. Select the **Review** tab; then select **Language** > Set **Proofing Language**.

**Comment by sylviefaulkner** [comment]
Hi @ValerieCle,
I agree with @kimviens that it is best to provide the "skip link" at the top of the page of the email to allow the users to skip to the other language. Many people add the text "English follows" or "French follows" but don't add the link. This is terrible for screen reader users but also lots of scrolling for long messages for a user that can't read the message in the language chosen to be the first. 

This is not mentioned anywhere in the [Making Accessible Emails](https://a11y.canada.ca/en/making-accessible-emails/) / but I find that the subject line should also be part of the body of the message because some are so long, that it gets cut off. 

Also, for a screen reader user, their default language may not be the first language used in the subject line so they're force to hear gibberish sounds which may bring confusion. When I give demo in my Department in relation to accessible email, I do recommend that the subject line be part of the heading 1 both the English message and the French message.

I would not recommend sending out 2 emails to accommodate the languages because I find users are already receive enough emails as it is without having to receive everything in double.

**Other barriers I notice in emails:**
- Contact information only available at the end of the message which can be missed by a user only reading the message in the first language.
- Message banner only displayed at the top of the page with an alt text either only in one language or in both language. An alt cannot be set to a specific language which forces a screen reader user to hear gibberish sounds which may bring confusion. I always recommend to include the banner in both languages.
- Using sub-lists can also become confusing for a screen reader users because some screen reader do not announce that the sub-list is a sub-list
- Using a table in emails is not like coding a table in an html document because we can not associate the table data to their table header which can cause confusion for screen reader users.
- When emails are focused more on the look than usability. Sometime being fancy creates overflowing content for users that required to increase the font.
